Added custom css class on edge labels

kaelhem opened the pull request:
Hi,
I've just made a small update on edge label to be able to specify custom css classes. 

It should fix https://github.com/dagrejs/dagre-d3/issues/195

tik-stbuehler commented:
For consistency (with `labelStyle` and `labelType`) you probably should name the label class property `labelClass` instead of `className`.

(And I have no idea why you had to redo the way the class is applied - didn't `classed` work with the calculated class list?)

kaelhem commented:
Hello. I changed the property name for labelClass as you asked. I replaced the "classed" stuff that allow to add/remove a specific class name to be able to specify several classes in the `className` parameter (like this : `customClass1 customClass2`). 

If you think there is a better way to do it, tell me I will update the PR.

tik-stbuehler commented:
I'm not a maintainer, just a user (and I'm interested in the feature too) :)

According to the API doc [`classed`](https://github.com/d3/d3-selection/blob/v1.4.1/README.md#selection_classed) should work with multiple classes (like `"label customClass1 customClass2"` - i.e. just pass `labelClass` instead of `"label"`) - but I'm no d3 expert, so perhaps I'm missing something here.

kaelhem commented:
ok :)
I will have look, not a d3 expert too :)

tbo47 commented:
I ported the project to es6 and I accept contributions!
https://github.com/tbo47/dagre-es
